docs/source/cpp/api/array.rst

```diff
@@ -110,3 +110,12 @@ Utilities
    :project: arrow_cpp
    :members:
    :undoc-members:
+
+.. _api-array-from-json-string:
+
+FromJSONString Helpers
+----------------------
+
+.. doxygengroup:: array-from-json-string
+   :content-only:
```

Comment:
This omits the [group description](http://crossbow.voltrondata.com/pr_docs/46180/cpp/api/array.html#fromjsonstring-helpers); I think it'd be better to include the description rather than leave that blank
```suggestion
```

alternatively/in addition this could link back to the explanation in `docs/source/cpp/arrays.rst`